<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color : rgb(227, 219, 249)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>1461</width>
      <height>1000</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1311</width>
      <height>801</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1800</width>
      <height>1000</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Главная</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Тестирование онлайн</string>
     </attribute>
     <widget class="QPushButton" name="button_start">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>510</y>
        <width>631</width>
        <height>131</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Perpetua</family>
        <pointsize>28</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color : lightgreen</string>
      </property>
      <property name="text">
       <string>Начать тестирование</string>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>1281</width>
        <height>361</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2"/>
     </widget>
     <widget class="QWidget" name="graph_widget" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>510</y>
        <width>631</width>
        <height>341</height>
       </rect>
      </property>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>150</y>
         <width>55</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </widget>
     <widget class="QLineEdit" name="lineEdit_6">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>470</y>
        <width>631</width>
        <height>24</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Perpetua</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color : rgb(199, 150, 213)</string>
      </property>
      <property name="text">
       <string>                       Видео с камеры с выделенными областями интереса</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>470</y>
        <width>631</width>
        <height>24</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Perpetua</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color : rgb(199, 150, 213)</string>
      </property>
      <property name="text">
       <string>                                                              Видео с камеры</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_7">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>840</y>
        <width>631</width>
        <height>24</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Perpetua</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color : rgb(199, 150, 213)</string>
      </property>
      <property name="text">
       <string>                                                              График пульса</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="pulse_box">
      <property name="geometry">
       <rect>
        <x>644</x>
        <y>111</y>
        <width>636</width>
        <height>359</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>1</x>
        <y>111</y>
        <width>636</width>
        <height>359</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Загрузить тестовый файл</string>
     </attribute>
     <widget class="QPushButton" name="start_test_file">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>510</y>
        <width>771</width>
        <height>431</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Onyx</family>
        <pointsize>36</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color : rgb(199, 150, 213)</string>
      </property>
      <property name="text">
       <string>Проанализировать файл</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="file_result">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>15</y>
        <width>611</width>
        <height>931</height>
       </rect>
      </property>
      <property name="acceptDrops">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color : rgb(199, 150, 213)</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <column>
       <property name="text">
        <string notr="true">Файл</string>
       </property>
       <property name="toolTip">
        <string notr="true"/>
       </property>
       <property name="whatsThis">
        <string notr="true"/>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="chose_file">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>771</width>
        <height>441</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Onyx</family>
        <pointsize>36</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color : rgb(199, 150, 213)</string>
      </property>
      <property name="text">
       <string>Выбрать тестовый файл</string>
      </property>
      <property name="icon">
       <iconset theme="myicon"/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Настройки</string>
     </attribute>
     <widget class="QPushButton" name="apply_settings">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>220</y>
        <width>1261</width>
        <height>151</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>36</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color : rgb(199, 150, 213)</string>
      </property>
      <property name="text">
       <string>Применить настройки</string>
      </property>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>10</y>
        <width>621</width>
        <height>181</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QFontComboBox" name="audio_device">
         <property name="font">
          <font>
           <family>Perpetua</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="currentText">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_3">
         <property name="font">
          <font>
           <family>Perpetua</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Входные данные для аудио</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>619</width>
        <height>181</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QFontComboBox" name="cam_device">
         <property name="font">
          <font>
           <family>Perpetua</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="currentText">
          <string>Neutral Face</string>
         </property>
         <property name="currentIndex">
          <number>178</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_4">
         <property name="font">
          <font>
           <family>Perpetua</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Входные данные для видео</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QTableWidget" name="table_probabilities_text">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1490</x>
      <y>20</y>
      <width>381</width>
      <height>331</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : rgb(165, 132, 255)</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <row>
     <property name="text">
      <string>positive</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>neutral</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>sad</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>disgust</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>other</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>angry</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>fear</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>other</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Вероятности распознанные по смысловой нагрузке</string>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="table_probabilities_audio">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1490</x>
      <y>360</y>
      <width>381</width>
      <height>221</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : rgb(165, 132, 255)</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <row>
     <property name="text">
      <string>neutral</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>angry</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>positive</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>sad</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>other</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Вероятности аудио </string>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="table_probabilities_video">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1490</x>
      <y>590</y>
      <width>381</width>
      <height>291</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::PreventContextMenu</enum>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : rgb(165, 132, 255)</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <row>
     <property name="text">
      <string>angry</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>disgust</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>fear</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>happy</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>sad</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>surprise</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>neutral</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Вероятности видео</string>
     </property>
    </column>
   </widget>
   <widget class="QLineEdit" name="resullt">
    <property name="geometry">
     <rect>
      <x>1530</x>
      <y>890</y>
      <width>301</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Perpetua</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Эмоция: None</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>26</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>99</horstretch>
     <verstretch>111</verstretch>
    </sizepolicy>
   </property>
   <widget class="QMenu" name="menuDetector_emotion">
    <property name="title">
     <string>Detector_emotion</string>
    </property>
   </widget>
   <addaction name="menuDetector_emotion"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
